demos/link-prediction/homogeneous-comparison-link-prediction.ipynb

```diff
@@ -0,0 +1,1515 @@
+{
```

Comment:
<p>The <code>&lt;a name=""&gt;</code>  links don't work on Read the Docs, which is our main venue for people to read the demos. You can explore this at https://stellargraph--1658.org.readthedocs.build/en/1658/demos/link-prediction/homogeneous-comparison-link-prediction.html .</p><p> </p><p>The HTML has two problems there:</p><p> </p><ul><li>the <code>#ref1</code>  (etc.) links don't go anywhere and are confusing</li><li>the HTML of the references themselves comes out funny: they're wrapped in a <code>&lt;blockquote&gt;</code>,  when they should just be normal text (it's why they're indented in the rendering on Read the Docs above, but not in the rendering in Jupyter)</li></ul><p> </p><p>I think it's fine to just leave them as non-links, and people can scroll manually if they absolutely want the citation (since this document is short enough and I imagine the citations will be relatively rarely followed). This means removing the  <code>[...](#...)</code> around each of the <code>[1]</code>  citations, as well as the <code>&lt;a ...&gt;&lt;/a&gt;</code>  pieces.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Thanks! I will remove the citation commands.</p>